### PR TITLE
Endpoints

### PR DESCRIPTION
**Pull request by adamgyulavari**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -30,4 +30,6 @@ dependencies {
 	runtime('mysql:mysql-connector-java')
 	compileOnly('org.projectlombok:lombok')
 	testCompile('org.springframework.boot:spring-boot-starter-test')
+	compile 'com.google.api-client:google-api-client:1.23.0'
+	compile( 'com.google.apis:google-api-services-calendar:v3-rev271-1.23.0')
```
Comment (adamgyulavari) [inline build.gradle]: either the first or the second syntax just not both

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/KalendaryoApplication.java [hunk 2]
```diff
@@ -3,6 +3,8 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import java.util.Date;
```
Comment (adamgyulavari) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/KalendaryoApplication.java]: this is unnecessary import

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controller/Controller.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package com.greenfoxacademy.opal.kalendaryo.kalendaryo.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.model.Event;
+
+@RestController()
```
Comment (adamgyulavari) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controller/Controller.java]: the parentheses are unnecessary

File: src/main/resources/application.properties.example [hunk 4]
```diff
@@ -1,8 +0,0 @@
-server.port=${OPAL_CAL_PORT}
-spring.datasource.url=jdbc:mysql://${RDS_HOSTNAME}:${RDS_PORT}/${RDS_DB_NAME}
-spring.datasource.username=${RDS_USERNAME}
-spring.datasource.password=${RDS_PASSWORD}
-spring.jpa.hibernate.ddl-auto=update
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
-spring.logging.level.org.hibernate.SQL=debug
-spring.jpa.show-sql=true
```
Comment (adamgyulavari) [inline src/main/resources/application.properties.example]: probably you will still in need of this on the long run

File: build.gradle [hunk 5]
```diff
@@ -30,4 +30,6 @@ dependencies {
 	runtime('mysql:mysql-connector-java')
 	compileOnly('org.projectlombok:lombok')
 	testCompile('org.springframework.boot:spring-boot-starter-test')
+	compile ('com.google.api-client:google-api-client:1.23.0')
```
[the remark this answers is not in the document]
Comment (nezihcihan) [reply build.gradle]: the space is removed

File: src/main/resources/application.properties.example [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+#API Configration
+google.client.client-id=${CLIENT_ID}
+google.client.client-secret=${CLIENT_SECRET}
+google.client.access-token-uri=https://www.googleapis.com/oauth2/v3/token
+google.client.user-authorization-uri=https://accounts.google.com/o/oauth2/auth?access_type=offline&prompt=consent
+google.client.client-authentication-scheme=query
+google.client.scope=profile,email,https://www.googleapis.com/auth/calendar
+google.resource.user-info-uri=https://www.googleapis.com/oauth2/v2/userinfo
+google.resource.prefer-token-info=true
+#google.client.redirectUri=http://localhost:9000/login/google
```
[the remark this answers is not in the document]
Comment (nezihcihan) [reply src/main/resources/application.properties.example]: added the extra line

**Pull request merged**